ngapConvert/AmfId.go

```diff
@@ -16,12 +16,12 @@ func AmfIdToNgap(amfId string) (regionId, setId, ptrId aper.BitString) {
 	setId = HexToBitString(amfId[2:5], 10)
 	tmpByte, err := hex.DecodeString(amfId[4:])
 	if err != nil {
-		logger.NgapLog.Warningln("AmfId From Models To NGAP Error: ", err.Error())
+		logger.NgapLog.Warnf("amfId From Models To NGAP Error: %v", err)
```

Comment:
I think this will fail in formatting or linting

Reply:
Why?

Reply:
I have seen you raising many PRs which makes first letter capital letter in the past.

Reply:
Actually, it has been the other way around (making first letter lower case to follow `golang` coding guidance/practices)

Reply:
> I think this will fail in formatting or linting

BTW, as you can see in the `lint` GHA (`golangci-lint` and `staticcheck`), PR successfully passes the checks